kernel/src/tty/types.rs

```diff
@@ -11,6 +11,7 @@ pub enum MalType {
     Symbol(String),
     List(Vec<MalType>),
     Bool(bool),
+    String(String),
```

Comment:
nit: Similarly, I don't know if `String` is the best name for an enum variant. What a lot of interpreters/compilers do is give them a slightly different name, ie [`krate` for crates](https://github.com/rust-lang/rust/blob/6b20506d17f4e5e5bf5bcad7e94add4d754b0ae3/compiler/rustc_ast_lowering/src/lib.rs#L316) in the Rust compiler so it doesn't mask a keyword. Maybe something like `MString`? Or `Str`? `LString` for Lisp string?

Reply:
You're right, but I really don't like the alternatives, so I think it's not too bad as long as I keep using MalType::String